### ISSUE TITLE
Missing shortcut in "About..." option in menu

### ISSUE THREAD

**Issue by samo79** [kas1e/Odyssey#28]
In most programs, including other software created by Fab there are always a shortcut to quicly open the about window... a shortcut in the about option menu.. 

Add "A+?" shortcut in the "About..." menu

**Comment by javierdlr** [comment]
Change:
C:\cygwin64\amiga\Odyssey\odyssey-r155188-1.23\Source\WebKit\OrigynWebBrowser\Api\MorphOS\appclass.cpp

	{ NM_ITEM , MENU(ABOUT)           , 0, 0, 0, (APTR)MNA_ABOUT },

to
	{ NM_ITEM , MENU(ABOUT)           ,(STRPTR) "?", 0, 0, (APTR)MNA_ABOUT },

haven't try it so far.

**Comment by samo79** [comment]
@javierdlr 

Yep it should works.. btw do you have access to the repo to applying direct changes?
